.github/workflows/pr.yml

```diff
@@ -18,7 +18,7 @@
 name: PR
 
 on:
-  pull_request:
+  pull_request_target:
```

Comment:
let's revert this. there is asf policy against using this. we can ignore the labeler issue for now.